dstep/translator/Type.d

```diff
@@ -350,9 +352,10 @@ body
         {
             if (children[dimension].kind == CXCursorKind.CXCursor_IntegerLiteral)
             {
+                auto token = tokenInsideSquareBrackets(cursor);
                 auto expansions = context.macroIndex.queryExpansion(children[dimension]);
 
-                if (expansions.length == 1)
+                if (expansions.length == 1 && token.kind == TokenKind.identifier)
```

Comment:
I was thinking about trying to create a reduced test case for this in C/C++ and ask on the Clang mailing list if the location is correct.